sdkproject/Assets/Mapbox/Examples/Scripts/LabelTextSetter.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Mapbox.Unity.MeshGeneration.Interfaces;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using System;
+
+public class LabelTextSetter : MonoBehaviour, IFeaturePropertySettable
+{
+	public void Set(Dictionary<string, object> props)
+	{
+		if (props.ContainsKey("name"))
+			GetComponentInChildren<TextMesh>().text = props["name"].ToString();
```

Comment:
TMP is still an additional asset you have to install afaik? and what happens if devs don't have it? would probably get errors I guess?

Reply:
Sure, but it's on the asset store and it's free (and owned by Unity). I don't see any issue with putting it directly into the SDK. I guess we could verify the license, but I imagine it will be bundled inside Unity editor in future versions anyway.

Reply:
my only problem is it'll pop errors and simply won't run. otherwise I would LOVE to have cinemachine and post processing stack inside as well.
It's perfectly possible some devs won't want/use TMP in their projects though and this would bloat their project.